scripts/download_amex_transactions.py

```
@@ -0,0 +1,377 @@
+    for partial in download_dir.glob("*.crdownload"):
+        if partial not in before:
+            partial.unlink(missing_ok=True)
+    raise RuntimeError("Timed out waiting for Chrome to finish the Amex CSV download")
+
+
+def move_download(download: Path, output_dir: Path) -> Path:
+    output_dir = output_dir.expanduser().resolve()
+    output_dir.mkdir(parents=True, exist_ok=True)
+    destination = output_dir / sanitize_filename(download.name)
+    if destination.exists():
+        raise FileExistsError(f"Refusing to overwrite existing file: {destination}")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)